src/binwalk/magic/firmware

```diff
@@ -468,11 +468,13 @@
 # Found in DIR-100 firmware
 0       string      AIH0        AIH0 firmware header, header size: 48,
 >12     ubelong     0           {invalid}
->12     ubelong+48  x           {size:%d}
+>12     ubelong     >0x4000000  {invalid}
+>12     ubelong     x           {size:%d}
```

Comment:
'been working on an extractror for the associated WDK-format, that's when I found that issue, might be a +48 too much in my code though. I find it unlikely that they'd make such a small change to the header-format.